r/configure

```diff
@@ -96,7 +96,7 @@ else
       echo "**** or retry with ARROW_USE_PKG_CONFIG=false"
     fi
   else
-    if [ "$UNAME" = "Darwin" ]; then
+    if [ "$UNAME" = "Darwin" ] && [ "$FORCE_TOOLS_LIBS_SCRIPT" = "" ]; then
```

Comment:
```suggestion
    if [ "$UNAME" = "Darwin" ] && [ "$FORCE_TOOLS_LIBS_SCRIPT" != "true" ]; then
```

Also see L38 above and do the same thing with this var to make it case insensitive